cleanlab_studio/internal/api/api.py

```diff
@@ -706,5 +713,70 @@ async def tlm_get_confidence_score(
     return cast(JSONDict, res_json)
 
 
+def enrich_data(
```

Comment:
i assume this is the helper method?
please clarify that in the docstring. I typically avoid using same method name for main method and the helper method, except in cases where they are all instance methods